keras_nlp/models/deberta_v3/deberta_v3_tokenizer.py

```diff
@@ -48,6 +48,11 @@ class DebertaV3Tokenizer(SentencePieceTokenizer):
             `bytes` object with a serialized SentencePiece proto. See the
             [SentencePiece repository](https://github.com/google/sentencepiece)
             for more details on the format.
+        mask_token_id: The token ID (int) of the mask token (`[MASK]`). If
```

Comment:
I think given that most users will not need an MLM task, we should actually make this optional when "brining your own data." Something like...

 - Use one of our presets. `self.mask_token_id` is set and works as expected.
 - Pass your own local copy of a deberta spm file and don't set anything. `self.mask_token_id` is `None`, everything works except `DebertaMaskedLM`, which throws a friendly error message. We can cover the error in #732
 - Optional (but already working here). Use your own custom spm file with a `"[MASK]"` token. `self.mask_token_id` is set and works as expected.

Does that make sense to you?

Reply:
(1) and (3) were already taken care of. Anyway, I've pushed changes which solves all three cases, and resolves the other comment.

Reply:
Thanks! Looks good. Left some thoughts below on how we could maybe make the subclass changes a bit easier by modifying the super class.